### ISSUE TITLE
Add tests for Unicode strings in predicate pushdown

### ISSUE THREAD

**Issue by morazow** [exasol/spark-exasol-connector#12]
Given a predicate condition from Spark with Unicode string value check that it correctly reads from Exasol.

#### UnitTest

```
EqualTo("str_col", "unicodestring") => WHERE str_col = 'unicodestring'
```

#### IntegrationTest

Create a table with a unicode values in Exasol docker and assert that we can read it correctly.

The integration test can be combined with other integration tests.

**Comment by jpizagno** [comment]
@morazow @3cham  
Can I take this ticket?  
I think it will be quick, because I am already familiar with the integration tests.

**Comment by morazow** [comment]
Fixed on #34
